db/seeds.rb

```diff
@@ -6,3 +6,14 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+emails = ['random1@gmail.com', 'random2@gmail.com', 'random3@gmail.com']
+3.times do
+teachers = Teacher.find_or_create_by!(email: emails.shift ) do |teacher|
```

Comment:
Hey, nice DRY! Although.. maybe you can do it even simpler using `emails.each`? :)

Also - I'm pretty sure that rubocop will have some offenses in this file 😅 I'm also a little bit surprised that your IDE hasn't autoformated this on save. Does your Visual Studio Code no longer run rubocop on save?

Reply:
Like this?